docs/user/installation-in-wordpress-multisite.md

```diff
@@ -0,0 +1,109 @@
+- Configure the components you wish to enable network-wide, such as Friends, Groups, Private Messaging, etc.
+
+### Network-wide Installation: BuddyPress Root Blog on a Secondary Site
+One way to set up BuddyPress network-wide is to activate it across the entire network. Still, a secondary site should be designated as the main hub for all BuddyPress content and activities. This setup can be especially helpful if the network's main site has a specific function and is not meant to be used for community interactions.
+
+#### Step 1: Identify Root Site and Modify Configuration
+
+1. **Access Network Admin Dashboard**:
+   - Navigate to `Dashboard → Network Admin`.
+
+2. **Identify the Secondary Site**:
+   - Click on the `Sites` link.
+   - Locate and note the ID number of the secondary site you want to use as the root for BuddyPress.
+
+3. **Configure wp-config.php**:
+   - Access your server via FTP or your hosting file manager to open the `wp-config.php` file.
+   - Add the following line, replacing `$blog_id` with the actual ID of your chosen site:
+     ```php
+     define('BP_ROOT_BLOG', $blog_id);
+     ```
+   - Save and close the file.
+
+#### Step 2: Install and Activate BuddyPress
+
+1. **Install BuddyPress**:
+   - Go back to the Network Admin dashboard.
+   - Navigate to `Plugins → Add New`.
+   - Search for "BuddyPress", then click `Install Now`.
+
+2. **Activate BuddyPress Network-Wide**:
+   - After installation, click `Activate`.
+   - Post activation, you will be directed to the BuddyPress Welcome screen, confirming the successful setup.
+
+#### Step 3: Configure BuddyPress for Multisite
+
+1. **Configure Network Settings**:
+   - In the Network Admin, navigate to `Settings → BuddyPress`.
+   - Set up and adjust the necessary settings and components, such as user profiles, groups, and site tracking, to suit your network’s needs.
+   - Confirm that all settings are properly saved and effectively applied network-wide, focusing on the newly designated root site.
+
+Following these steps, you can successfully configure BuddyPress on a secondary site within your WordPress Multisite network, centralizing all BuddyPress activities on your chosen site and preserving the main site's primary functionalities.
+
+### Setting Up BuddyPress on an Individual Site in WordPress Multisite
+
+This guide provides detailed steps for configuring BuddyPress to operate on an individual site within a WordPress Multisite network. This allows for site-specific community features without affecting other sites in the network.
+
+#### Step 1: Install BuddyPress
+
+1. **Access Site Admin Dashboard**:
+   - Navigate to the site dashboard where you want to activate BuddyPress.
+
+2. **Install BuddyPress**:
+   - Go to `Plugins → Add New`.
+   - Search for "BuddyPress", then click `Install Now`.
+   - Do not activate it yet network-wide. Only proceed to activation in the individual site’s admin area.
```

Comment:
This step should be explained into the [BuddyPress Installation Guide for Website Administrators](https://github.com/buddypress/buddypress/blob/master/docs/user/getting-started/activate-buddypress.md)

```suggestion

```